pkg/clusteragent/admission/mutate/config/config.go

```diff
@@ -65,29 +65,37 @@ const (
 
 // Webhook is the webhook that injects DD_AGENT_HOST and DD_ENTITY_ID into a pod
 type Webhook struct {
-	name            string
-	isEnabled       bool
-	endpoint        string
-	resources       []string
-	operations      []admissionregistrationv1.OperationType
-	mode            string
-	wmeta           workloadmeta.Component
-	injectionFilter mutatecommon.InjectionFilter
-	datadogConfig   config.Component
+	name              string
+	isEnabled         bool
+	endpoint          string
+	resources         []string
+	operations        []admissionregistrationv1.OperationType
+	mode              string
+	wmeta             workloadmeta.Component
+	injectionFilter   mutatecommon.InjectionFilter
+	localServiceName  string
+	traceAgentSocket  string
+	dogStatsDSocket   string
+	socketPath        string
+	typeSocketVolumes bool
```

Comment:
maybe for readability, you can split the list of field with an empty line before, or create a sub structure `type config struct`

Reply:
Thank you for the comment. I totally agree this struct was getting a little out of hand.